### PR TITLE
Troubleshooting faq

### PR DESCRIPTION
**Pull request by nomadscientist**
I've added single-cell matrix group optionality to the troubleshooting FAQ and then pushed it into a bunch of tutorials and our general single cell site

### CODE REVIEW

File: topics/single-cell/faqs/single_cell_omics.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+Did you know we have a unique Single Cell Lab with all our single cell tools highlighted to make it easier to use on Galaxy? We recommend this site for all your single cell analysis needs, particularly for newer users.
 
-This **subdomain** is uses the main European Galaxy infrastructure and power, it's just organised better for users of particular analyses...like single cell!
+The Single Cell Lab currently uses the main European Galaxy infrastructure and power, it's just organised better for users of particular analyses...like single cell!
 
-Try it out! All your histories/workflows/logins from the general [European Galaxy instance](https://usegalaxy.eu) will be there!
+Try it out! All your histories/workflows/logins from the general [European Galaxy server](https://usegalaxy.eu) will be there!
 
- - <i class="fa fa-tv" aria-hidden="true"></i> [Single cell omics Galaxy instance](https://singlecell.usegalaxy.eu)
+ - <i class="fa fa-tv" aria-hidden="true"></i> [Single Cell Lab](https://singlecell.usegalaxy.eu)
```
Comment (hexylena) [inline topics/single-cell/faqs/single_cell_omics.md:15]: could you add this to `_config.yml` as a named icon please?

Comment (nomadscientist) [reply topics/single-cell/faqs/single_cell_omics.md]: yes ma'am!

File: _config.yml [hunk 2]
```diff
@@ -194,6 +194,7 @@ icon-tag:
   slides: fab fa-slideshare
   solution: far fa-eye
   sticky-note: fas fa-sticky-note
+  subdomain: fa fa-tv
```
[the remark this answers is not in the document]
Comment (hexylena) [reply _config.yml:197]:
```suggestion
  subdomain: fa fa-tv
  galaxy-lab: fa fa-tv
```

**Review by hexylena — DISMISSED** [review]

File: _config.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (nomadscientist) [reply _config.yml:198]: smart!

File: _config.yml [hunk 3]
```diff
@@ -194,6 +195,8 @@ icon-tag:
...
+  galaxy-lab: fa fa-tv
```
Comment (hexylena) [inline _config.yml:199]:
```suggestion
```

(since you added it as well in the other palce)

**Pull request merged**